properties/P000168.md

```diff
@@ -1,11 +1,22 @@
 ---
 uid: P000168
-name: $\omega$C
+name: Countable sets are discrete
 aliases:
 - Countable sets are closed
+- Cid
+- $\omega$C
 refs:
+- mathse: 4995501
+  name: Answer to "Does every door space with an infinite compact set have a single non-isolated point?"
+- zb: "0665.54003"
+  name: On spaces whose denumerable subspaces are discrete.
 - mathse: 4751818
   name: Answer to What separation is required to ensure extremally disconnected spaces are sequentially discrete?
 ---
 
-Countable subsets are closed.
+Every {P57} subspace is {P52}.
```

Comment:
I assume `{P57}` is better than saying countable in words because by clicking this, readers will know that we mean "(finite or infinite) countable".

Reply:
That and I just like cross-referencing: want to know what else pi-Base knows about countable spaces? Here's a handy link.